build.gradle

```diff
@@ -189,6 +189,7 @@ project(':ambry-api') {
         compile "org.codehaus.jackson:jackson-core-asl:$jacksonVersion"
         compile "org.codehaus.jackson:jackson-mapper-asl:$jacksonVersion"
         compile "io.netty:netty-buffer:$nettyVersion"
+        compile "io.netty:netty-all:$nettyVersion"
```

Comment:
let's try to see if we can avoid adding the netty-all dependency to ambry-api. If we cannot avoid it, we can remove the dependencies on netty-buffer, since netty-all includes it already.

Reply:
I have reverted the change in this file as it's no longer needed. I am passing SSLSession instead of ChannelHandlerContext in the validateConnection() interface.